blocks/v2-truck-features/v2-truck-features.css

```diff
@@ -0,0 +1,207 @@
+.section.v2-truck-features-container > .v2-truck-features-wrapper {
+  padding: 0;
+}
+
+.v2-truck-features {
+  padding: 0;
+  color: var(--text-color);
+}
+
+.v2-truck-features__content {
+  width: 100%;
+  height: calc(100vh - var(--nav-height) - var(--inpage-navigation-height));
+  position: sticky;
+  top: calc(var(--nav-height) + var(--inpage-navigation-height));
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.v2-truck-features__heading {
+  font-family: var(--ff-subheadings-medium);
+  font-size: 32px;
+  line-height: var(--headline-3-line-height);
+  margin: 0;
+  padding: 40px 16px;
+}
+
+.v2-truck-features__slides {
+  margin: 0;
+  list-style: none;
+  padding: 0;
+}
+
+.v2-truck-features ul ul:first-of-type {
+  display: none;
+}
+
+.v2-truck-features .v2-truck-features__images-list {
+  margin: 0;
+  padding: 0;
+}
+
+.v2-truck-features .v2-truck-features__images-list li {
+  display: none;
+  width: auto;
+}
+
+.v2-truck-features .v2-truck-features__slide {
+  display: none;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.v2-truck-features .v2-truck-features__slide.v2-truck-features__slide--active {
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.v2-truck-features .v2-truck-features__slide h4 {
+  font-family: var(--ff-subheadings-medium);
+  line-height: var(--headline-5-line-height);
+  font-size: 18px;
+  letter-spacing: 0.36px;
+  margin: 0;
+}
+
+.v2-truck-features .v2-truck-features__slide h4::before {
+  width: 2px;
+  height: 27px;
+  content: '';
+  display: block;
+  position: absolute;
+  background: var(--c-primary-white);
+  left: 16px;
+}
+
+.v2-truck-features .v2-truck-features__slide p {
+  margin: 0;
+}
+
+.v2-truck-features .v2-truck-features__slide--active h4::before {
+  background: var(--c-accent-red);
+}
+
+.v2-truck-features
+  .v2-truck-features__images-list
+  li.v2-truck-features__slide-image--active {
+  display: flex;
+  justify-content: center;
+}
+
+.v2-truck-features__slide-image--active picture,
+.v2-truck-features__slide-image--active img {
+  display: flex;
+  height: 280px;
+  aspect-ratio: 375/275;
```

Comment:
I took it from mobile design.
<img width="1792" alt="obraz" src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/125962117/4a7515e1-d604-455d-a7f5-583d5ee9e23d">
Because the images are different sizes and positioned in different ways I took the image frame aspect-ratio.

Reply:
It is updated